sensor/DNSProxyPlugin.js

```diff
@@ -149,7 +156,22 @@ class DNSProxyPlugin extends Sensor {
   }
   
   async globalOn() {
-    this.launchServer();
+    sclient.subscribe(BF_SERVER_MATCH)
+    sclient.on("message", async (channel, message) => {
+      
+      switch(channel) {
+        case BF_SERVER_MATCH:
+          let msgObj;
+          try {
+            msgObj = JSON.parse(message)
+          } catch(err) {
+            log.error("parse msg failed", err, message)
+          }
```

Comment:
Check if MAC address exists in devices list before invoking processRequest. Dnsmasq may generate FF:FF:FF:FF:FF:FF for unknown IPv4 address